code/components/nui-core/src/NUIInitialize.cpp

```diff
@@ -445,13 +451,23 @@ void Component_RunPreInit()
 #endif
 
 	// CEF keeps loading/unloading this - load it ourselves to make the refcount always 1
-	LoadLibrary(L"bluetoothapis.dll");
+	auto bluetoothapisHandle = LoadLibrary(L"bluetoothapis.dll");
+	(void)(bluetoothapisHandle); // it's ok if there is an error
 
 	// load Chrome dependencies ourselves so that the system won't try loading from other paths
-	LoadLibrary(MakeRelativeCitPath(L"bin/chrome_elf.dll").c_str());
-	LoadLibrary(MakeRelativeCitPath(L"bin/libEGL.dll").c_str());
+	auto chromeElfHandle = LoadLibrary(MakeRelativeCitPath(L"bin/chrome_elf.dll").c_str());
+	if(!chromeElfHandle)
+		MessageBoxW(NULL, L"Could not load bin/chrome_elf.dll.", L"CitizenFX", MB_ICONWARNING | MB_OK);

```

Comment:
Users will typically ignore message boxes opened in sequence - this is already a bit of a concern with Ldr errors at this time, but when explicitly making error messages one _should not_ open multiple dialog boxes, since the user will only screenshot the last one.

Reply:
Totally understandable. But since there is no unified way in the codebase of tracing errors I just took as example code few lines below with messagebox error. Could you give me the cue how to log this please?